wcivf/apps/elections/tests/test_election_views.py

```diff
@@ -39,3 +40,45 @@ def test_election_detail_view(self):
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response, "elections/election_view.html")
         self.assertContains(response, self.election.nice_election_name)
+
+class ElectionPostViewTests(TestCase):
+    def setUp(self):
+        self.election = ElectionFactory(
+            name="Adur local election",
+            election_date="2021-05-06",
+            slug="local.adur.churchill.2021-05-06",
+        )
+        self.post = PostFactory()
+        self.post_election = PostElectionFactory(election=self.election, post=self.post)
+        people = [PersonFactory() for p in range(5)]
```

Comment:
I would move this block (lines 53-60) to within your `test_num_candidates` test. Otherwise your `test_zero_candidates` test will fail, as it will have 5 candidates rather than 0.